dev-docs/publisher-api-reference.md

```diff
@@ -1008,6 +1008,12 @@ Set a global bidder timeout:
 pbjs.setConfig({ bidderTimeout: 3000 });
 {% endhighlight %}
 
+{: .alert.alert-warning :}
+**Bid Timeouts and JavaScript Timers**  
+Note that it's still possible for a bid to get into the auction that responds after this timeout. This is due to how [`setTimeout`](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout) works in JS: it queues the callback in the event loop in an approximate location that *should* execute after this time but *it is not guaranteed*.  
```

Comment:
Looks good. Slight modification required here `...get into the auction that responds after this timeout...` technically the timeout is executing late, the bid should not get into the auction _after_ the timeout, the timeout just can get triggered later than expected.

Reply:
So would it be correct to state "Note that it's possible for the timeout to be triggered later than expected, leading to a bid participating in the auction later than expected" ?

Reply:
Yeah that makes sense.

Reply:
Cool, just pushed that update.